README.md

```diff
@@ -359,10 +335,6 @@ u := "https://192-168-0-101.nip.io:543/auth/realms/test/protocol/openid-connect/
   * After importing the client configurations from the `fapi-conformance-suite-configs` directory, you may need to replace the URLs of
   the keycloak server like using `as.keycloak-fapi.org` with your publicly available Keycloak instance like `https://85-244-72-90.nip.io:543/auth/...`.
   Same applies for the resource server URL, which is `rs.keycloak-fapi.org` in the configuration files by default, and needs to be replaced with your publicly available one.
```

Comment:
I would change this line to:
Same applies for the resource server URL, which is `rs.keycloak-fapi.org` and CIBA approval endpoint, which is `aes.keycloak-fapi.org` in the configuration files by default. These needs to be replaced with your publicly available one.

Reply:
I've fixed.